redis/commands/core.py

```diff
@@ -1767,18 +1767,32 @@ def pexpire(self, name: KeyT, time: ExpiryT) -> ResponseT:
             time = int(time.total_seconds() * 1000)
         return self.execute_command("PEXPIRE", name, time)
 
-    def pexpireat(self, name: KeyT, when: AbsExpiryT) -> ResponseT:
-        """
-        Set an expire flag on key ``name``. ``when`` can be represented
-        as an integer representing unix time in milliseconds (unix time * 1000)
-        or a Python datetime object.
+    def pexpireat(
+            self,
+            name: KeyT,
+            when: AbsExpiryT,
+            option: str = None
```

Comment:
@dogukanteber WDYT about making these options (NX, XX, etc) variables in the function header. IMHO it would watch our other functions.